libnetdata/config/appconfig.c

```diff
@@ -819,23 +819,25 @@ void appconfig_generate(struct config *root, BUFFER *wb, int only_changed)
                   "#\n"
                   "\n# global netdata configuration\n");
 
-    for(i = 0; i <= 13 ;i++) {
+    for(i = 0; i <= 15 ;i++) {
         appconfig_wrlock(root);
         for(co = root->first_section; co ; co = co->next) {
             if(!strcmp(co->name, CONFIG_SECTION_GLOBAL))                 pri = 0;
-            else if(!strcmp(co->name, CONFIG_SECTION_HOST_LABEL))        pri = 1;
-            else if(!strcmp(co->name, CONFIG_SECTION_SQLITE))            pri = 2;
-            else if(!strcmp(co->name, CONFIG_SECTION_CLOUD))             pri = 3;
-            else if(!strcmp(co->name, CONFIG_SECTION_ML))                pri = 4;
-            else if(!strcmp(co->name, CONFIG_SECTION_HEALTH))            pri = 5;
-            else if(!strcmp(co->name, CONFIG_SECTION_STREAM))            pri = 6;
-            else if(!strcmp(co->name, CONFIG_SECTION_WEB))               pri = 7;
-            // by default, new sections will get pri = 8 (set at the end, below)
-            else if(!strcmp(co->name, CONFIG_SECTION_REGISTRY))          pri = 9;
-            else if(!strcmp(co->name, CONFIG_SECTION_GLOBAL_STATISTICS)) pri = 10;
-            else if(!strcmp(co->name, CONFIG_SECTION_STATSD))            pri = 11;
-            else if(!strcmp(co->name, CONFIG_SECTION_PLUGINS))           pri = 12;
-            else if(!strncmp(co->name, "plugin:", 7))                    pri = 13; // << change the loop too if you change this
+            else if(!strcmp(co->name, CONFIG_SECTION_DIRECTORIES))       pri = 1;
+            else if(!strcmp(co->name, CONFIG_SECTION_LOGS))              pri = 2;
+            else if(!strcmp(co->name, CONFIG_SECTION_HOST_LABEL))        pri = 3;
+            else if(!strcmp(co->name, CONFIG_SECTION_SQLITE))            pri = 4;
+            else if(!strcmp(co->name, CONFIG_SECTION_CLOUD))             pri = 5;
+            else if(!strcmp(co->name, CONFIG_SECTION_ML))                pri = 6;
+            else if(!strcmp(co->name, CONFIG_SECTION_HEALTH))            pri = 7;
+            else if(!strcmp(co->name, CONFIG_SECTION_STREAM))            pri = 8;
+            else if(!strcmp(co->name, CONFIG_SECTION_WEB))               pri = 9;
+            // by default, new sections will get pri = 10 (set at the end, below)
+            else if(!strcmp(co->name, CONFIG_SECTION_REGISTRY))          pri = 11;
+            else if(!strcmp(co->name, CONFIG_SECTION_GLOBAL_STATISTICS)) pri = 12;
+            else if(!strcmp(co->name, CONFIG_SECTION_STATSD))            pri = 13;
+            else if(!strcmp(co->name, CONFIG_SECTION_PLUGINS))           pri = 14;
+            else if(!strncmp(co->name, "plugin:", 7))                    pri = 15; // << change the loop too if you change this
             else pri = 8; // this is used for any new (currently unknown) sections
```

Comment:
Line `835` says that `will get pri = 10 (set at the end, below)`, but you still kept `pri = 8`, please, change it for `10`.